config/src/lib.rs

```diff
@@ -1839,6 +1839,17 @@ impl Config {
         // across the win32/wsl boundary
         let mut wsl_env = std::env::var("WSLENV").ok();
 
+        // Since we are running inside the appimage, we have "$APPIMAGE"
+        // and "$APPDIR" set inside the wezterm appimage. This gets
+        // propagated to the child processes. Since some apps (including
+        // wezterm) use these variables to detect if they are running in
+        // an appimage, the child processes get misconfigured
+        // We should just unset them too.
```

Comment:
```suggestion
        // If we are running as an appimage, we will have "$APPIMAGE"
        // and "$APPDIR" set in the wezterm process. These will be
        // propagated to the child processes. Since some apps (including
        // wezterm) use these variables to detect if they are running in
        // an appimage, those child processes will be misconfigured.
        // Ensure that they are unset.
```